tldr.py

```diff
@@ -504,6 +508,40 @@ def main() -> None:
                 with open(command, encoding='utf-8') as open_file:
                     output(open_file.read().encode('utf-8').splitlines(),
                            plain=options.markdown)
+    elif options.search:
+        command = '-'.join(options.command)
+        page = None
+        maxprob = 0
+        searchquery = options.search.split(' ')
+
+        for i in get_commands():
+            if i.startswith(command):
+                for p in get_platform_list():
+                    result = load_page_from_cache(i, p, options.language)
+                    if result is not None and have_recent_cache(i, p, options.language):
+                        break
+                try:
+                    result = result.decode("utf-8")
+                    result = result.split()
+                except AttributeError:
+                    print("Please update cache")
+                    quit()
```

Comment:
Per the note in [the docs](https://docs.python.org/3/library/constants.html#constants-added-by-the-site-module) for anything coming from site module:

> They are useful for the interactive interpreter shell and should not be used in programs.

Just raise a `SystemExit`.